src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -1,8 +1,12 @@
 package frc.robot.flywheel;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.FlywheelConstants;
 
+import javax.naming.ldap.Control;
```

Comment:
I think this could also be removed since it is unused, and it would help to clean up the code.